staging/cse/windows/debug/update-scripts.ps1

```diff
@@ -94,3 +94,12 @@ $collectLogsScript=($collectLogsScript | Where { $_ -notlike '*DownloadFile*' })
 # Remove Invoke-WebRequest in collectlogs.ps1
 $collectLogsScript=($collectLogsScript | Where { $_ -notlike '*Invoke-WebRequest*' })
 $collectLogsScript | Set-Content collectlogs.ps1
+
+# Update startpacketcapture.ps1
+Write-Host "************************************************************" 
+Write-Host "Please manually remove Get-WebFile and the download steps in"
```

Comment:
should line 83 - line 88 be deleted?

Reply:
no. this script is used to download windows debug scripts from microsoft/SDN. AKS caches these debug scripts in all AKS Windows nodes and these scripts should never download anything from internet.